### DISCUSSION TITLE
Crowdin translation project, is it ready?

### DISCUSSION THREAD

**Discussion by kevinlin18** [dbeaver/dbeaver#12921] [DBeaver Development]
Readme didn't mention it, is it ready for translators?
Strings keeps updated, but no one translation there
I want to help translate dbeaver into Chinese Traditional(Taiwan) which added in 21.1.1,

**Comment by LonwoLonwo** [comment]
Hello @kevinlin18 
Unfortunately, it is not ready for now. Maybe in the future.
Now you can help us with the translation directly. Read more about it here https://github.com/dbeaver/dbeaver/wiki/Localization

**Reply by kevinlin18** [reply]
Got it, thank you
